bin/offline-cluster.sh

```diff
@@ -47,9 +47,7 @@ ansible-playbook -i $INVENTORY_FILE $ANSIBLE_DIR/seed-offline-containerd.yml
 ansible-playbook -i $INVENTORY_FILE $ANSIBLE_DIR/sync_time.yml -v
 
 # Run the rest of kubespray. This should bootstrap a kubernetes cluster successfully:
-ansible-playbook -i $INVENTORY_FILE $ANSIBLE_DIR/kubernetes.yml --skip-tags bootstrap-os,preinstall,container-engine
-
-./bin/fix_default_router.sh
+ansible-playbook -i $INVENTORY_FILE $ANSIBLE_DIR/kubernetes.yml --skip-tags bootstrap-os,preinstall,container-engine,multus
```

Comment:
multius?

Reply:
@julialongtin I'm surprised as you are, but there are traces of *multus* in our code base: https://github.com/kubernetes-sigs/kubespray/blob//docs/multus.md?plain=1#L1

Me needs a <scratches-head> emoji.

Reply:
this multus network plugin playbooks were being run while installing k8s, even the variable to install multus was set to false and was throwing some dependency issues, hence added that tag in the skip list